step-definitions/auth-authorized.steps.ts

```diff
@@ -0,0 +1,93 @@
+import { AxiosResponse } from '@nestjs/terminus/dist/health-indicator/http/axios.interfaces'
+import { AxiosError } from '@nestjs/terminus/dist/errors/axios.error'
+import { binding, given, then, when } from 'cucumber-tsflow'
+import { assert } from 'chai'
+import nacl from 'tweetnacl'
+import dotenv from 'dotenv'
+import axios from 'axios'
+import bs58 from 'bs58'
+
+dotenv.config()
+
+@binding()
+export class AuthAuthorizedSteps {
+  private serverHost = process.env.HOST
+  private serverPort = process.env.PORT
+  private result: AxiosResponse | null = null
+  private error: AxiosError | null = null
+  private apiLink = ''
+
+  private async login(): Promise<string> {
+    let cookie = ''
+
+    try {
+      const requestMessage = await axios({
+        method: 'GET',
+        url: this.serverHost + ':' + this.serverPort + '/api/auth/requestMessage',
```

Comment:
Use [template litera](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)l to create a string